src/Twig/FrontendMenuExtension.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Bolt\Twig;
+
+use Bolt\Menu\CachedFrontendMenuBuilder;
+use Twig\Environment;
+use Twig\Extension\AbstractExtension;
+use Twig\TwigFunction;
+
+class FrontendMenuExtension extends AbstractExtension
+{
+    /** @var CachedFrontendMenuBuilder */
+    private $menuBuilder;
+
+    public function __construct(CachedFrontendMenuBuilder $menuBuilder)
+    {
+        $this->menuBuilder = $menuBuilder;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFunctions(): array
+    {
+        $safe = ['is_safe' => ['html']];
+        $env = ['needs_environment' => true];
+
+        return [
+            new TwigFunction('menu', [$this, 'buildMenu'], $env + $safe),
+            new TwigFunction('menu_json', [$this, 'buildMenuJson']),
+        ];
+    }
+
+    public function buildMenu(Environment $twig, ?string $name = null, string $template = '_sub_menu.twig', string $class = '', bool $withsubmenus = true): string
```

Comment:
getMenu

Reply:
Seriously? First you want me to change them from `getMenu` to `buildMenu`, and now you want me to change them back?

Reply:
No, I wanted to change it only in Builder class. Here it was ok :)

Reply:
Updated!